params/protocol_params.go

```diff
@@ -149,14 +149,17 @@ func GetMaximumExtraDataSize() uint64 {
 type CodeFormat uint8
 
 const (
-	CodeFormatEVM CodeFormat = iota
+	CodeFormatEVM                   CodeFormat = iota // contracts deployed before IstanbulCompatible are assigned CodeFormatEVM
```

Comment:
```suggestion
	CodeFormatEVMConstantinople                   CodeFormat = iota // contracts deployed before IstanbulCompatible are assigned CodeFormatEVM
```

Reply:
https://github.com/klaytn/klaytn/commit/ac19b0238823c7c8e7982b89e4197415472dfc12 reflected

Reply:
This is reverted.